docs/docs/integrations.md

```diff
@@ -34,5 +34,68 @@ fancy2.path
 #> PosixPath('mydir/myfile.txt')
 ```
 
+
+## Pandas and [pandas-path](https://github.com/drivendataorg/pandas-path)
+
+[`pandas-path`](https://github.com/drivendataorg/pandas-path) provides `pathlib` functions through as custom accessor (`.path`). The library also supports registering custom accessors for any class that implements that pathlib API.
+
+### The `.cloud` accessor
+
+We expose a `.cloud` accessor on `pandas.Series` and `pandas.Index` objects if you import it. This allows you to access any `CloudPath` method or property directly on a Series by just adding `.cloud`.
+
+To use the `.cloud` accessor, you must have `pandas-path` installed through pip:
+
+```
+pip install pandas-path
+```
+
+All you need to do to register the accessor is `from cloudpathlib.pandas import cloud`.
```

Comment:
Why the extra module layer, instead of just `import cloudpathlib.pandas`?

Reply:
This is to mirror the the pandas-path import which looks like `from pandas_path import path`. I think it's nice to (1) make the structure similar, and (2) see the name of the accessor when you do the import so you can see how they might get tied together.